aspnet/mvc/overview/older-versions-1/security/index.md

```diff
@@ -9,7 +9,7 @@ ms.assetid: 346c6b44-0ff4-49af-ac94-6e74d952b4eb
 msc.legacyurl: /mvc/overview/older-versions-1/security
 msc.type: chapter
 ---
-# Security
+# Password Protection
```

Comment:
What about we change this line to: "Security (Password Protection)"

That way we keep the author's heading there, and we make it unique with your addition.